### DISCUSSION TITLE
Weapon type "whip" missing

### DISCUSSION THREAD

**Discussion by Warbier89** [JWinsen/DSA5-Foundry-VTT-Char2PDF#5] [Ideas]
Hello and thanks for the great module.

We noticed when exporting a hero that the weapon type "whips" is not integrated and therefore there is an error when exporting.
![image](https://user-images.githubusercontent.com/15850207/157842465-2fb20e0c-3767-44b8-be11-8ddcf837db95.png)

With kind regards
Warbier89

**Comment by JWinsen** [comment]
Ja, das ist korrekt. Ich habe das Problem nicht mit mit den Peitschen, sondern mit allen Sonderfertigkeiten die durch DSA Erweiterungen implementiert werden. Es müsste ein neues und komplettes PDF Template geschaffen werden. Da das ein ständiges Nacharbeiten bedeuteten würde, gibt es nur zwei Möglichkeiten. 

1. Entweder man baut das PDF dynamisch 
2. Man greift auf andere Möglichkeiten zurück. 

Das Erstere gestaltet sich nicht gerade leicht wenn man auch noch ein entsprechend ansehnliches Bild (PDF) als Resultat zu haben. Zu Nummer 2 habe ich gerade etwas in der Erprobung. Ist ziemlich simple aber man nutzt die Daten wie sie bereits im HTML (also wie in Foundry) dargestellt werden. 

Wenn Du das Modul vorab ebenfalls versuchen möchtest, sage bitte Bescheid. 

Grüße 
Jens

**Reply by Warbier89** [reply]
Hi Jens,
ich verstehe natürlich die Problematik.

Mein "Workarround" war es jetzt folgende Zeile im Code zu ergänzen:
![image](https://user-images.githubusercontent.com/15850207/157854217-ff64f20d-bb6a-4bdc-a047-301acecc54b6.png)

Da es sich dabei nur um eine Alternative Lösung handelt, würde mich natürlich auch das andere Modul Interessieren von dem du schreibst 😉.

Grüße
Michele
